app/views/users/index.html.erb

```diff
@@ -1,22 +1,19 @@
-<h1>Users#index</h1>
-<p>Find me in app/views/users/index.html.erb</p>
-<!-- app/views/users/index.html.erb -->
-<h1>All Users</h1>
-<p>Here is a list of all users:</p>
-<!-- Implement code to display the list of users -->
-
-<!-- app/views/users/show.html.erb -->
-<h1>User Details</h1>
-<p>Here are the details for the user:</p>
-<!-- Implement code to display the details of a single user -->
-
-<!-- app/views/posts/index.html.erb -->
-<h1>All Posts</h1>
-<p>Here is a list of all posts by the user:</p>
-<!-- Implement code to display the list of posts for a given user -->
-
-<!-- app/views/posts/show.html.erb -->
-<h1>Post Details</h1>
-<p>Here are the details for the post:</p>
-<!-- Implement code to display the details of a single post -->
-
+<h1 class='title'>Welcome to Blog-Haven</h1>
+<% if @users.empty? %>
+  <p class='user_empty-message'>No users yet!</p>
+<% else %>
+  <div class='container'>
+    <% @users.each do |user| %>
+      <div>
+        <div class='avatar'>
+            <%= image_tag(user.photo) %>
+          </div>
+          <div><h2><%= user.name %></h2></div>
+          <div> <p><%= link_to "Number of posts: x", user_path(user) %></p> </div>
+         
+         
+        
+      </div>
```

Comment:
- [ ] As the previous reviewer mentioned, the post count is not showing, and also you are not matching the wireframe given
your design :point_down: 
![Screenshot from 2023-07-21 23-28-06](https://github.com/eudondian/Blog-Haven/assets/66097371/bf00bed9-12ce-401b-9526-5710495d3fff)
given design :point_down: 
![Screenshot from 2023-07-21 23-35-02](https://github.com/eudondian/Blog-Haven/assets/66097371/9130b59f-b4d5-4050-b7e8-0676ab774b86)

Reply:
well resolved! ✔️
Design follows the wireframe though there's not bother applied to design.